### ISSUE TITLE
2024-12-17, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#23]
# Today's Arxiv Papers

(1) [SepLLM: Accelerate Large Language Models by Compressing One Segment into One Separator](http://arxiv.org/abs/2412.12094v1), 2024-12-16

(2) [Instruction-based Image Manipulation by Watching How Things Move](http://arxiv.org/abs/2412.12087v1), 2024-12-16

(3) [CPath-Omni: A Unified Multimodal Foundation Model for Patch and Whole Slide Image Analysis in Computational Pathology](http://arxiv.org/abs/2412.12077v1), 2024-12-16

(4) [CG-Bench: Clue-grounded Question Answering Benchmark for Long Video Understanding](http://arxiv.org/abs/2412.12075v1), 2024-12-16

(5) [Making FETCH! Happen: Finding Emergent Dog Whistles Through Common Habitats](http://arxiv.org/abs/2412.12072v1), 2024-12-16

(6) [Can LLM Prompting Serve as a Proxy for Static Analysis in Vulnerability Detection](http://arxiv.org/abs/2412.12039v1), 2024-12-16

(7) [LLMs for Cold-Start Cutting Plane Separator Configuration](http://arxiv.org/abs/2412.12038v1), 2024-12-16

(8) [SpeechPrune: Context-aware Token Pruning for Speech Information Retrieval](http://arxiv.org/abs/2412.12009v1), 2024-12-16

(9) [The Open Source Advantage in Large Language Models (LLMs)](http://arxiv.org/abs/2412.12004v1), 2024-12-16

(10) [LLM-RG4: Flexible and Factual Radiology Report Generation across Diverse Input Contexts](http://arxiv.org/abs/2412.12001v1), 2024-12-16

(11) [Combining Large Language Models with Tutoring System Intelligence: A Case Study in Caregiver Homework Support](http://dx.doi.org/10.1145/3706468.3706516), 2024-12-16

(12) [ExecRepoBench: Multi-level Executable Code Completion Evaluation](http://arxiv.org/abs/2412.11990v1), 2024-12-16

(13) [SciFaultyQA: Benchmarking LLMs on Faulty Science Question Detection with a GAN-Inspired Approach to Synthetic Dataset Generation](http://arxiv.org/abs/2412.11988v1), 2024-12-16

(14) [Cost-Effective Label-free Node Classification with LLMs](http://arxiv.org/abs/2412.11983v1), 2024-12-16

(15) [DARWIN 1.5: Large Language Models as Materials Science Adapted Learners](http://arxiv.org/abs/2412.11970v1), 2024-12-16

(16) [Inferring Functionality of Attention Heads from their Parameters](http://arxiv.org/abs/2412.11965v1), 2024-12-16

(17) [Advancing Comprehensive Aesthetic Insight with Multi-Scale Text-Guided Self-Supervised Learning](http://arxiv.org/abs/2412.11952v1), 2024-12-16

(18) [OpenReviewer: A Specialized Large Language Model for Generating Critical Scientific Paper Reviews](http://arxiv.org/abs/2412.11948v1), 2024-12-16

(19) [Precise Length Control in Large Language Models](http://arxiv.org/abs/2412.11937v1), 2024-12-16

(20) [A Survey of Mathematical Reasoning in the Era of Multimodal Large Language Model: Benchmark, Method & Challenges](http://arxiv.org/abs/2412.11936v1), 2024-12-16

(21) [Stepwise Reasoning Error Disruption Attack of LLMs](http://arxiv.org/abs/2412.11934v1), 2024-12-16

(22) [PICLe: Pseudo-Annotations for In-Context Learning in Low-Resource Named Entity Detection](http://arxiv.org/abs/2412.11923v1), 2024-12-16

(23) [RetroLLM: Empowering Large Language Models to Retrieve Fine-grained Evidence within Generation](http://arxiv.org/abs/2412.11919v1), 2024-12-16

(24) [Does VLM Classification Benefit from LLM Description Semantics?](http://arxiv.org/abs/2412.11917v1), 2024-12-16

(25) [CharacterBench: Benchmarking Character Customization of Large Language Models](http://arxiv.org/abs/2412.11912v1), 2024-12-16

(26) [Can Language Models Rival Mathematics Students? Evaluating Mathematical Reasoning through Textual Manipulation and Human Experiments](http://arxiv.org/abs/2412.11908v1), 2024-12-16

(27) [PunchBench: Benchmarking MLLMs in Multimodal Punchline Comprehension](http://arxiv.org/abs/2412.11906v1), 2024-12-16

(28) [Using Instruction-Tuned Large Language Models to Identify Indicators of Vulnerability in Police Incident Narratives](http://arxiv.org/abs/2412.11878v1), 2024-12-16

(29) [GeoX: Geometric Problem Solving Through Unified Formalized Vision-Language Pre-training](http://arxiv.org/abs/2412.11863v1), 2024-12-16

(30) [A Theory of Formalisms for Representing Knowledge](http://arxiv.org/abs/2412.11855v1), 2024-12-16

(31) [Towards Understanding Systems Trade-offs in Retrieval-Augmented Generation Model Inference](http://arxiv.org/abs/2412.11854v1), 2024-12-16

(32) [A Benchmark and Robustness Study of In-Context-Learning with Large Language Models in Music Entity Detection](http://arxiv.org/abs/2412.11851v1), 2024-12-16

(33) [A Distributed Collaborative Retrieval Framework Excelling in All Queries and Corpora based on Zero-shot Rank-Oriented Automatic Evaluation](http://arxiv.org/abs/2412.11832v1), 2024-12-16

(34) [Are You Doubtful? Oh, It Might Be Difficult Then! Exploring the Use of Model Uncertainty for Question Difficulty Estimation](http://arxiv.org/abs/2412.11831v1), 2024-12-16

(35) [EventSum: A Large-Scale Event-Centric Summarization Dataset for Chinese Multi-News Documents](http://arxiv.org/abs/2412.11814v1), 2024-12-16

(36) [UAlign: Leveraging Uncertainty Estimations for Factuality Alignment on Large Language Models](http://arxiv.org/abs/2412.11803v1), 2024-12-16

(37) [No More Adam: Learning Rate Scaling at Initialization is All You Need](http://arxiv.org/abs/2412.11768v1), 2024-12-16

(38) [IDEA-Bench: How Far are Generative Models from Professional Designing?](http://arxiv.org/abs/2412.11767v1), 2024-12-16

(39) [QUENCH: Measuring the gap between Indic and Non-Indic Contextual General Reasoning in LLMs](http://arxiv.org/abs/2412.11763v1), 2024-12-16

(40) [Harnessing Language for Coordination: A Framework and Benchmark for LLM-Driven Multi-Agent Control](http://arxiv.org/abs/2412.11761v1), 2024-12-16

(41) [CSR:Achieving 1 Bit Key-Value Cache via Sparse Representation](http://arxiv.org/abs/2412.11741v1), 2024-12-16

(42) [Personalized LLM for Generating Customized Responses to the Same Query from Different Users](http://arxiv.org/abs/2412.11736v1), 2024-12-16

(43) [LLMs Can Simulate Standardized Patients via Agent Coevolution](http://arxiv.org/abs/2412.11716v1), 2024-12-16

(44) [Seeker: Towards Exception Safety Code Generation with Intermediate Language Agents Framework](http://arxiv.org/abs/2412.11713v1), 2024-12-16

(45) [MiMoTable: A Multi-scale Spreadsheet Benchmark with Meta Operations for Table Reasoning](http://arxiv.org/abs/2412.11711v1), 2024-12-16

(46) [CoinMath: Harnessing the Power of Coding Instruction for Math LLMs](http://arxiv.org/abs/2412.11699v1), 2024-12-16

(47) [On Large Language Models in Mission-Critical IT Governance: Are We Ready Yet?](http://arxiv.org/abs/2412.11698v1), 2024-12-16

(48) [Ultra-High-Definition Dynamic Multi-Exposure Image Fusion via Infinite Pixel Learning](http://arxiv.org/abs/2412.11685v1), 2024-12-16

(49) [Multimodal LLM for Intelligent Transportation Systems](http://arxiv.org/abs/2412.11683v1), 2024-12-16

(50) [LLM-DaaS: LLM-driven Drone-as-a-Service Operations from Text User Requests](http://arxiv.org/abs/2412.11672v1), 2024-12-16

(51) [C3oT: Generating Shorter Chain-of-Thought without Compromising Effectiveness](http://arxiv.org/abs/2412.11664v1), 2024-12-16

(52) [Private Yet Social: How LLM Chatbots Support and Challenge Eating Disorder Recovery](http://arxiv.org/abs/2412.11656v1), 2024-12-16

(53) [QPruner: Probabilistic Decision Quantization for Structured Pruning in Large Language Models](http://arxiv.org/abs/2412.11629v1), 2024-12-16

(54) [Fool Me, Fool Me: User Attitudes Toward LLM Falsehoods](http://arxiv.org/abs/2412.11625v1), 2024-12-16

(55) [VG-TVP: Multimodal Procedural Planning via Visually Grounded Text-Video Prompting](http://arxiv.org/abs/2412.11621v1), 2024-12-16

(56) [EvoLlama: Enhancing LLMs' Understanding of Proteins via Multimodal Structure and Sequence Representations](http://arxiv.org/abs/2412.11618v1), 2024-12-16

(57) [MT-LENS: An all-in-one Toolkit for Better Machine Translation Evaluation](http://arxiv.org/abs/2412.11615v1), 2024-12-16

(58) [SPaR: Self-Play with Tree-Search Refinement to Improve Instruction-Following in Large Language Models](http://arxiv.org/abs/2412.11605v1), 2024-12-16

(59) [Aligning Visual and Semantic Interpretability through Visually Grounded Concept Bottleneck Models](http://arxiv.org/abs/2412.11576v1), 2024-12-16

(60) [The Role of Natural Language Processing Tasks in Automatic Literary Character Network Construction](http://arxiv.org/abs/2412.11560v1), 2024-12-16

(61) [Enhancing Healthcare Recommendation Systems with a Multimodal LLMs-based MOE Architecture](http://arxiv.org/abs/2412.11557v1), 2024-12-16

(62) [Token Prepending: A Training-Free Approach for Eliciting Better Sentence Embeddings from LLMs](http://arxiv.org/abs/2412.11556v1), 2024-12-16

(63) [Towards a Speech Foundation Model for Singapore and Beyond](http://arxiv.org/abs/2412.11538v1), 2024-12-16

(64) [Let your LLM generate a few tokens and you will reduce the need for retrieval](http://arxiv.org/abs/2412.11536v1), 2024-12-16

(65) [ON as ALC: Active Loop Closing Object Goal Navigation](http://arxiv.org/abs/2412.11523v1), 2024-12-16

(66) [DART: An AIGT Detector using AMR of Rephrased Text](http://arxiv.org/abs/2412.11517v1), 2024-12-16

(67) [Glimpse: Enabling White-Box Methods to Use Proprietary Models for Zero-Shot LLM-Generated Text Detection](http://arxiv.org/abs/2412.11506v1), 2024-12-16

(68) [Embodied CoT Distillation From LLM To Off-the-shelf Agents](http://arxiv.org/abs/2412.11499v1), 2024-12-16

(69) [FTP: A Fine-grained Token-wise Pruner for Large Language Models via Token Routing](http://arxiv.org/abs/2412.11494v1), 2024-12-16

(70) [NoteContrast: Contrastive Language-Diagnostic Pretraining for Medical Text](http://arxiv.org/abs/2412.11477v1), 2024-12-16

(71) [Understanding Knowledge Hijack Mechanism in In-context Learning through Associative Memory](http://arxiv.org/abs/2412.11459v1), 2024-12-16

(72) [Towards Better Multi-task Learning: A Framework for Optimizing Dataset Combinations in Large Language Models](http://arxiv.org/abs/2412.11455v1), 2024-12-16

(73) [ACE-$M^3$: Automatic Capability Evaluator for Multimodal Medical Models](http://arxiv.org/abs/2412.11453v1), 2024-12-16

(74) [Whisper-GPT: A Hybrid Representation Audio Large Language Model](http://arxiv.org/abs/2412.11449v1), 2024-12-16

(75) [Towards Scientific Discovery with Generative AI: Progress, Opportunities, and Challenges](http://arxiv.org/abs/2412.11427v1), 2024-12-16

(76) [ConceptEdit: Conceptualization-Augmented Knowledge Editing in Large Language Models for Commonsense Reasoning](http://arxiv.org/abs/2412.11418v1), 2024-12-16

(77) [RL-LLM-DT: An Automatic Decision Tree Generation Method Based on RL Evaluation and LLM Enhancement](http://arxiv.org/abs/2412.11417v1), 2024-12-16

(78) [Leveraging Retrieval-Augmented Tags for Large Vision-Language Understanding in Complex Scenes](http://arxiv.org/abs/2412.11396v1), 2024-12-16

(79) [Temporal Contrastive Learning for Video Temporal Reasoning in Large Vision-Language Models](http://arxiv.org/abs/2412.11391v1), 2024-12-16

(80) [INTERACT: Enabling Interactive, Question-Driven Learning in Large Language Models](http://arxiv.org/abs/2412.11388v1), 2024-12-16

(81) [How Can LLMs and Knowledge Graphs Contribute to Robot Safety? A Few-Shot Learning Approach](http://arxiv.org/abs/2412.11387v1), 2024-12-16

(82) [Why Does ChatGPT "Delve" So Much? Exploring the Sources of Lexical Overrepresentation in Large Language Models](http://arxiv.org/abs/2412.11385v1), 2024-12-16

(83) [FinLoRA: Finetuning Quantized Financial Large Language Models Using Low-Rank Adaptation](http://arxiv.org/abs/2412.11378v1), 2024-12-16

(84) [ChatTime: A Unified Multimodal Time Series Foundation Model Bridging Numerical and Textual Data](http://arxiv.org/abs/2412.11376v1), 2024-12-16

(85) [Codenames as a Benchmark for Large Language Models](http://arxiv.org/abs/2412.11373v1), 2024-12-16

(86) [Can AI Extract Antecedent Factors of Human Trust in AI? An Application of Information Extraction for Scientific Literature in Behavioural and Computer Sciences](http://arxiv.org/abs/2412.11344v1), 2024-12-16

(87) [Zero-Shot Prompting Approaches for LLM-based Graphical User Interface Generation](http://arxiv.org/abs/2412.11328v1), 2024-12-15

(88) [RoLargeSum: A Large Dialect-Aware Romanian News Dataset for Summary, Headline, and Keyword Generation](http://arxiv.org/abs/2412.11317v1), 2024-12-15

(89) [Reliable, Reproducible, and Really Fast Leaderboards with Evalica](http://arxiv.org/abs/2412.11314v1), 2024-12-15

(90) [Sequence-Level Analysis of Leakage Risk of Training Data in Large Language Models](http://arxiv.org/abs/2412.11302v1), 2024-12-15

(91) [CATER: Leveraging LLM to Pioneer a Multidimensional, Reference-Independent Paradigm in Translation Quality Evaluation](http://arxiv.org/abs/2412.11261v1), 2024-12-15

(92) [Do Tutors Learn from Equity Training and Can Generative AI Assess It?](http://dx.doi.org/10.1145/3706468.3706531), 2024-12-15

(93) [Beyond Discrete Personas: Personality Modeling Through Journal Intensive Conversations](http://arxiv.org/abs/2412.11250v1), 2024-12-15

(94) [TrimLLM: Progressive Layer Dropping for Domain-Specific LLMs](http://arxiv.org/abs/2412.11242v1), 2024-12-15

(95) [Smaller Language Models Are Better Instruction Evolvers](http://arxiv.org/abs/2412.11231v1), 2024-12-15

(96) [Task-Oriented Dialog Systems for the Senegalese Wolof Language](http://arxiv.org/abs/2412.11203v1), 2024-12-15

(97) [Drawing the Line: Enhancing Trustworthiness of MLLMs Through the Power of Refusal](http://arxiv.org/abs/2412.11196v1), 2024-12-15

(98) [From Votes to Volatility Predicting the Stock Market on Election Day](http://arxiv.org/abs/2412.11192v1), 2024-12-15

(99) [Leveraging Large Language Models for Active Merchant Non-player Characters](http://arxiv.org/abs/2412.11189v1), 2024-12-15

(100) [Cultural Palette: Pluralising Culture Alignment via Multi-agent Palette](http://arxiv.org/abs/2412.11167v1), 2024-12-15
